packages/connect-form/src/components/advanced-options-tabs/ssh-tunnel-tab/proxy-and-ssh-tunnel-tab.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { ChangeEvent, useState, useCallback } from 'react';
+import { ConnectionOptions } from 'mongodb-data-service';
+import {
+  Label,
+  RadioBox,
+  RadioBoxGroup,
+  spacing,
+  css,
+} from '@mongodb-js/compass-components';
+import ConnectionStringUrl from 'mongodb-connection-string-url';
+import { MongoClientOptions } from 'mongodb';
+
+import { UpdateConnectionFormField } from '../../../hooks/use-connect-form';
+import {
+  SSHConnectionOptions,
+  TunnelType,
+} from '../../../utils/connection-ssh-handler';
+
+import SshTunnelIdentity from './ssh-tunnel-identity';
+import SshTunnelPassword from './ssh-tunnel-password';
+import Socks from './socks';
+import { ConnectionFormError } from '../../../utils/validation';
+
+interface TabOption {
+  id: string;
+  title: string;
+  type: TunnelType;
+  component: React.FC<{
+    sshTunnelOptions?: SSHConnectionOptions;
+    updateConnectionFormField: UpdateConnectionFormField;
+    connectionStringUrl: ConnectionStringUrl;
+    errors: ConnectionFormError[];
+  }>;
+}
+
+const options: TabOption[] = [
+  {
+    title: 'None',
+    id: 'none',
+    type: 'none',
+    component: function None() {
+      return <></>;
+    },
+  },
+  {
+    title: 'Use Password',
+    id: 'password',
+    type: 'password',
+    component: SshTunnelPassword,
+  },
+  {
+    title: 'Use Identity File',
+    id: 'identity',
+    type: 'identity',
+    component: SshTunnelIdentity,
+  },
+  {
+    title: 'Socks',
```

Comment:
I think it would be a bit less ambiguous if we used the same titles that we have in the [figma files](https://www.figma.com/file/Bs5CaG0UNgxoXO6AtuGsCd/Compass_Connection?node-id=437%3A8620) for this, because neither “password” nor “identity file” very clearly imply that they refer to SSH – for example, I did not understand that “password tab” referred to SSH while reviewing the tests – and because we should be clear about the fact that Socks does not refer to either Socks4 or Socks5 but instead only Socks5:

```suggestion
    title: 'SSH with Password',
    id: 'password',
    type: 'password',
    component: SshTunnelPassword,
  },
  {
    title: 'SSH with Identity File',
    id: 'identity',
    type: 'identity',
    component: SshTunnelIdentity,
  },
  {
    title: 'Socks5',
```

Reply:
actually these were the titles when this was implemented. however new titles are more accurate.